### DISCUSSION TITLE
Set the cell location in a worksheet

### DISCUSSION THREAD

**Discussion by ergins23** [dfinke/ImportExcel#1598] [Q&A]
I want to set the active cell of a worksheet after editing the worksheet so that it is selected when I open the sheet in Excel. 

I cannot find a function that does this. Any help would be greatly appreciated.

**Comment by dfinke** [comment]
hmm, don't think I have one. I'm not sure if it can be done thru the Excel object with this. There is a `-Activate` switch on the `Export-Excel` function, which is for setting the active sheet.
